app/precompiled.py

```diff
@@ -709,12 +709,23 @@ def _extract_text_from_page(page, rect):
     """
     words = page.get_text_words()
     mywords = [w for w in words if fitz.Rect(w[:4]).intersects(rect)]
+
+    def _get_address_from_get_textwords():
+        return page.get_text(clip=rect).strip()
+
     mywords.sort(key=itemgetter(-3, -2, -1))
     group = groupby(mywords, key=itemgetter(3))
     extracted_text = []
-    for _y1, gwords in group:
+    for _y2, gwords in group:
```

Comment:
why are we changing `_y1` to `_y2` here?

Reply:
it more accurately describes the actual variable we're grouping on  from the tuple (x1, y1, x2, y2, word, para_num, line_num, word_num)

Reply:
ok, should we maybe explain in the commit that we are changing it - so it doesn't seem random?